### PR TITLE
#6232 update forms as services section

### PR DESCRIPTION
**Pull request by backbone87**
this is to reflect recent changes to form type handling since symfony 2.8

| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.8+ |
| Fixed tickets | #6232 |

### CODE REVIEW

**Comment by backbone87** [comment]
since this is my first PR, was it correct PR against 2.8 branch? (since this is the version that introduced the change)

**Comment by WouterJ** [comment]
:+1: I like this.

And yes, perfect PR against the good branch.

File: best_practices/forms.rst [hunk 1]
```diff
@@ -71,13 +71,9 @@ Registering Forms as Services
...
-But this is *not* recommended unless you plan to reuse the new form type in many
-places or embed it in other forms directly or via the
-:doc:`CollectionType </reference/forms/types/collection>`.
-
-For most forms that are used only to edit or create something, registering
-the form as a service is over-kill, and makes it more difficult to figure
-out exactly which form class is being used in a controller.
+This is only needed if your form type requires some dependencies to be injected
+by the container, otherwise it is just unnecessary overhead and therefore
+*not* recommended to do this for each and every form type class.
```
Comment (WouterJ) [inline best_practices/forms.rst]: A very minor thing is that for some reason, people hate reading docs. So I think we can improve this sentence a little bit by removing some not-needed words like replacing "just unnecessary overhead" with "unnecessary overhead" and "each and every form type class" with "for all form type classes".

If you want to apply these changes, you can navigate to your fork ( http://github.com/backbone87/symfony-docs ), select the branch of this PR (`patch-1`) and edit the file. If you, for any reason, do not want to do it, we will apply these changes while merging the PR.

**Comment by xabbuh** [comment]
Thank you very much @backbone87 for not only catching this mistake in the documentation but for also creating the PR to fix it. I have merged your changes into the `2.8` branch (I squashed your commits during merging that's why your PR is shown as closed, but you will still get the credit for your contribution) from where they will be merged into all higher branches.

**Pull request closed without merging**